playwright.config.ts

```diff
@@ -0,0 +1,96 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+// See: https://playwright.dev/docs/test-configuration.
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  timeout: 30 * 1000,
+  expect: {
+    timeout: 5000
+  },
+
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+
+  /* See: https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* See: https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    actionTimeout: 0,
+    trace: 'on-first-retry',
+
+    /* See: https://playwright.dev/docs/test-advanced */
+    baseURL: 'http://localhost:4280/home',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
```

Comment:
will this one also cover Edge as its chromium based? or do we need the branded ones for that?

Reply:
Default Chromium option runs against latest version of Chromium and is recommended (valid for Chrome and Edge future proofing) but if required to run against specific versions (or stock versions) then configuration can be updated
https://playwright.dev/docs/browsers#google-chrome--microsoft-edge